pyproject.toml

```diff
@@ -2,6 +2,7 @@
 requires = [
   "setuptools>=42",
   "setuptools-git-versioning<2",
+  "versioneer[toml]==0.29",
```

Comment:
Don't we want to write it as `"versioneer[toml]>=0.29,<1.0"`?

Reply:
changed